ImageKitSample/Program.cs

```diff
@@ -0,0 +1,426 @@
+                OverlayTextPadding(40).
+                OverlayTextInnerAlignment("left").
+                OverlayRadius(10).
+                Progressive(true).
+                Lossless(true).
+                Trim(5).
+                Metadata(true).
+                ColorProfile(true).
+                DefaultImage("folder/file.jpg/"). //trailing slash case
+                Dpr(3).
+                EffectSharpen(10).
+                EffectUsm("2-2-0.8-0.024").
+                EffectContrast(true).
+                EffectGray().
+                Original().
+                Raw("h-200,w-300,l-image,i-logo.png,l-end")
+                ;
+
+            string imageURL = imagekit.Url(trans).Path(path).TransformationPosition("query").Generate();			
+           
+            Console.WriteLine("Generated image URL - {0}", imageURL);
+
+            ///// Generating Signed URL
+            var imgURL1 = "https://ik.imagekit.io/demo/default-image.jpg";
+            string[] queryParams = { "b=123", "a=test" };
+            try
+            {
+                var signedUrl = imagekit.Url(new Transformation().Width(400).Height(300))
+                .Src(imgURL1)
+                .QueryParameters(queryParams)
+                .ExpireSeconds(600)
+                .Signed()
+                .Generate();
+                Console.WriteLine("Signed Url for first image transformed with height: 300, width: 400: - {0}", signedUrl);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            #endregion
+
+            #region  Upload BY URI| Bytes | Base64
+
+            // Upload By URI
+            FileCreateRequest request = new FileCreateRequest
+            {
+                Url = new Uri(@"http://www.google.com/images/logos/ps_logo2.png"),
+                FileName = "test.jpg",
+            };
+            Result resp1 = imagekit.Upload(request);
+
+            // Upload by bytes
+            byte[] bytes = System.IO.File.ReadAllBytes(@"image file path");
+
+            FileCreateRequest ob = new FileCreateRequest
+            {
+                Bytes = bytes,
+                FileName = Guid.NewGuid().ToString(),
+            };            
+            List<string> tags = new List<string>
+            {
+                "Software",
+                "Developer",
+                "Engineer"
+            };
+            ob.Tags = tags;
+            ob.Folder = "demo1";
+            string customCoordinates = "10,10,20,20";
+            ob.CustomCoordinates = customCoordinates;
+            List<string> responseFields = new List<string>
+            {
+                "thumbnail",
+                "tags",
+                "customCoordinates"
+            };
+
+            ob.ResponseFields = responseFields;
+           
+
+            List<Extension> model1=new List<Extension>();
+            BackGroundImage bck=new BackGroundImage();
+            bck.name = "remove-bg";
```

Comment:
![image](https://user-images.githubusercontent.com/16056543/186336176-e59b7ffa-c129-40df-af32-6cd8c538290a.png)

Reply:
This is still not working.